entries/z/zeplin.io.json

```
@@ -0,0 +1,12 @@
+{
+  "Zeplin": {
+    "domain": "zeplin.io",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.zeplin.io/en/articles/6992335-getting-started-with-mfa",
```

```suggestion
    "documentation": "https://support.zeplin.io/en/articles/7189128",
```

I would recommend using this support article because it is for individual users and includes a link to the enterprise plan's configuration as well.